### ISSUE TITLE
Unable to Pull or Delete Project

### ISSUE THREAD

**Issue by timholmes** [aws-amplify/amplify-hosting#2974]
I have a project in  Amplify and haven't used it for a while (about a year).  I went in to connect to it and do a "pull".  It was unable to do a pull.  Finally since it was stuck I tried to just delete the project.  It is saying that it cannot find the s3 bucket or the .zip (assume that is for deployments).  How do I get this project cleaned up now?   (I hope this doesn't happen to one of my more crucial applications)

**Comment by hloriana** [comment]
Hi @timholmes 👋🏽 apologies for the delay in response. Please try running this AWS CLI command to delete your application:
`aws amplify delete-backend-environment --app-id <value> --environment-name <value>`. This will delete the backend environment resources. You should then be able to delete the frontend of your application from the console. Please let us know if this unblocks you. Thank you!

**Comment by timholmes** [comment]
@hloriana   yes I was able to get unblocked.  I created a new application and copied over the configs.  I'll try to delete the bad one with this suggestion when I get a min.
